python/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Instead of copying this here, would suggest symlinking to the top-level `LICENSE`. Python build tools will still pick it up. Plus it will save us some duplication. FWIW have done this in PR ( https://github.com/rapidsai/rmm/pull/1151 ) if it is helpful

Reply:
Agreed, I was planning on doing that as a final round of cleanup once we've finalized the workflow.

Reply:
@jakirkham I just tested making the change you did in #1151 and pip wheel doesn't seem to pick it up correctly. The license inserted into the wheel only contains the literal `../LICENSE`. Do we need to specify anything additional?

Reply:
Vyas and I discussed this offline. The GitHub diffs are confusing unfortunately. It's an actual symlink.

Reply:
This is done now.